tests/roles/backend_services/tasks/main.yaml

```diff
@@ -11,6 +11,20 @@
     cd {{ install_yamls_path }}
     make input
 
+- name: get service passwords
+  delegate_to: "{{ adoption_ospdo_host }}"
+  ansible.builtin.shell: |
+     oc get secret tripleo-passwords -o jsonpath='{.data.*}' | base64 -d>~/tripleo-standalone-passwords.yaml
+  when: ospdo_src is defined
```

Comment:
It is defined always. Both in minimal make target(false) and in ospdo target. It would be better to use :
when: ospdo_src| default(false) | bool

Reply:
I removed the definition from tes-minimal , so this would work as is